sdk/keyvault/keyvault-keys/README.md

```diff
@@ -40,6 +40,18 @@ To quickly create the needed Key Vault resources in Azure and to receive a conne
 
 [![](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-sdk-for-js%2Fmaster%2Fsdk%2Fkeyvault%2Ftest-resources.json)
 
+To fill this form:
+
+1. If you know about ARM templates, or if you're interested to know about them, we recommend you to take some time to read through the ARM template we are using for this button [here](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-sdk-for-js%2Fmaster%2Fsdk%2Fkeyvault%2Ftest-resources.json). You can read more about the concept of an ARM template in our [Intro to Microsoft Azure Resource Manager Templates (Video)](https://dev.to/azure/intro-to-microsoft-azure-resource-manager-templates-video-9cc).
```

Comment:
I'm assuming the [here] link is the same thing as the deploy button above, but it feels weird to call it out with such an unhelpful name as 'here', I think 'above' and not having it be linked would suffice? Either that or actually repeat the button wholesale

Reply:
Oh my bad, the here button is intended to go directly to the source of the ARM template, not to the deployment form.

Reply:
Fixed this! Please read it again.